Assets/Scripts/Auction/BiddingPlayer.cs

```diff
@@ -20,12 +23,48 @@ public class BiddingPlayer : NetworkBehaviour
     private Color maxChipColor;
     [SerializeField]
     protected TMP_Text signCross;
-    protected BiddingPlatform currentPlatform = null;
+    private BiddingPlatform _currentPlatform = null;
+    protected BiddingPlatform currentPlatform
+    {
+        get => _currentPlatform;
+        set
+        {
+            _currentPlatform = value;
+            instantiatedMaterials
+                .ForEach(material => 
+                    material.SetFloat("_DitherDensity", value != null ? 0.9f : 1f));
+        }
+    }
+    [SerializeField]
+    protected SkinnedMeshRenderer[] playerRenderers;
+    [SerializeField]
+    protected MeshRenderer signRenderer;
```

Comment:
I think SkinnedMeshRenderer and MeshRenderer both inherit from Renderer if you wanna merge these properties (no biggie though)

Reply:
Yeah, too annoying so I fixed